src/TelemetryServer.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express')
+const client = require('prom-client')
+
+class TelemetryServer {
+  constructor (logger = { info: console.log }) {
```

Comment:
**suggestion:** Given that there is no type interface it could be good to have docstring here. But it could just be an improvement later.